src/view_controls/view.py

```diff
@@ -132,20 +129,12 @@ def handle_events(self):
                 elif event.key == K_F4 and pygame.key.get_mods() & KMOD_ALT:
                     return Event.DONE
                 elif event.key == K_c and pygame.key.get_mods() & KMOD_CTRL:
-                    # Debug function to write the state to a json file
```

Comment:
Ouch, this looks like a revert of @Zamiell's ea12728ba700d1cc11c5a59c1c7f7b6a66143d17, and would remove the copy to clipboard feature.